src/main/java/com/devcycle/sdk/server/local/protobuf/CustomDataValue.java

```diff
@@ -0,0 +1,773 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Is there a build command that generates these? Can we make sure it's documented?

Reply:
i documented it and then removed the documentation as I figured out how to get gradle to do the code generation